SPECS/apache-commons-logging/apache-commons-logging.spec

```
@@ -40,6 +40,7 @@ BuildRequires:  javapackages-local-bootstrap
 BuildRequires:  junit
 Requires:       java >= 1.8
 Provides:       jakarta-%{short_name} = %{version}-%{release}
+Provides:       mvn(commons-logging:commons-logging)
```

Let's also add the version info to the `Provides`. Otherwise, if we ever have a different package providing the same capability, this one might be still taken instead. If you set a version, you later use `Obsoletes: [package] < [version]`.
```suggestion
Provides:       mvn(commons-logging:commons-logging) = %{version}-%{release}
```

And yes, we don't do this in many specs. We should, though.

---

Updated